src/git.js

```diff
@@ -15,12 +15,12 @@
  */
 
 const {Owner, createOwnersMap} = require('./owner');
-const bb = require('bluebird');
 const child_process = require('child_process');
 const yaml = require('yamljs');
 const path = require('path');
-const exec = bb.promisify(child_process.exec);
-const fs = bb.promisifyAll(require('fs'));
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
```

Comment:
Optional: Not sure if the methods in https://github.com/ampproject/amphtml/blob/master/build-system/exec.js can be reused here.

Reply:
i dont believe i need it for now